.github/workflows/main.yml

```diff
@@ -55,6 +55,10 @@ jobs:
         run: |
           set -euo pipefail
           go test -p 1 -json ./... -covermode=count -coverprofile=coverage.out 2>&1 | tee test_results.json
+      - name: Sanitize test results
+        # We're utilizing `tee` above which can capture non-json stdout output, we also need to remove non-json lines.
```

Comment:
```suggestion
        # We're utilizing `tee` above which can capture non-json stdout output so we need to remove non-json lines before parsing it with jq.
```

Reply:
@Olshansk rephrased it a bit to reflect this is necessary for the external github action.